src/java.base/share/classes/java/lang/foreign/Linker.java

```diff
@@ -195,6 +196,18 @@
  *     <td style="text-align:center;">{@link MemorySegment}</td>
  * </tbody>
  * </table></blockquote>
+ * <p>
+ * Due to limited ABI specification coverage, all the native linker implementations limit the function
```

Comment:
```suggestion
 * All the native linker implementations support function function descriptors that contain only so-called ...
```

Reply:
So, just drop the 'Due to limited ABI specification coverage'? I guess it's not really needed.